packages/ui/src/utilities/renderTable.tsx

```diff
@@ -195,9 +200,38 @@ export const renderTable = ({
     } as Column)
   }
 
+  if (!orderableFieldName) {
+    return {
+      columnState,
+      // key is required since Next.js 15.2.0 to prevent React key error
+      Table: <Table appearance={tableAppearance} columns={columnsToUse} data={docs} key="table" />,
```

Comment:
Is it possible we could have more than on Table with the same key since it is hard coded to "table"?

Reply:
I don't know how it relates to the key inside an Array.map which requires uniqueness, but at least in the case of joining fields with multiple tables on the same page it didn't give any errors.